io.github.dosbox-staging.yml

```diff
@@ -28,6 +28,9 @@ modules:
   # Build FluidSynth for General MIDI emulation
   - shared-modules/linux-audio/fluidsynth2.json
 
+  # Build SDL2 with libdecor for Wayland support
+  - shared-modules/SDL2/SDL2-2.26.1-with-libdecor.json
+
   # Build iir1 DSP filter lib
   - name: iir1
     buildsystem: cmake
```

Comment:
Could you change this to `cmake-ninja`, because the build-linter gives are warning on that. I know that it's not your code, but we might just as well fix it right now

Reply:
I have seen you already did in the PR #22.

Reply:
Yes this was fixed in another PR.